src/services/dashboard.js

```diff
@@ -14,6 +17,22 @@ function parseMonths(datastoreConfig, index, selectedIndices) {
   return indices;
 }
 
+function scrollUntilEmpty(datastoreConfig, data) {
+  const endpoint = `${datastoreConfig.elasticsearch}/_search/scroll?scroll=1m`;
+  const allData = data;
+
+  if (allData.hits.total !== allData.hits.hits.length) {
+    const scroll = request.post(`${endpoint}&scroll_id=${allData._scroll_id}`);
+    scroll.then(response => {
+      allData._scroll_id = response._scroll_id;
+      allData.hits.total = response.hits.total;
+      allData.hits.hits = [...allData.hits.hits, ...response.hits.hits];
```

Comment:
For certain queries this could explode browser memory.  We might want to add some kind of circuit breaker in the future.